sirepo/sim_data/raydata.py

```diff
@@ -30,5 +29,10 @@ def _input_files():
                 if v:
                     yield cls.lib_file_name_with_model_field('inputFiles', k, v)
         return [
-            data.models.analysisAnimation.notebook,
+            # data.models.scans.catalogName + '.zip',
+            SimData.raydata_notebook_zip_filename(data),
```

Comment:
A more pythonic way of calling this since we are inside of the class is to say `cls.raydata_notebook_zip_filename(data)`

Reply:
bd781ab41c48e24e796b4abc76b8e738de72ba2d